client/src/components/chess-game/ChessGame.js

```diff
@@ -65,6 +65,10 @@ export default class ChessGame extends React.Component {
     }
     this.startGame();
     receiveMove(moveData => this.onMoveReceived(moveData));
+    opponentDisconnected(() => {
+      disconnect();
+      this.endGame(GAME_DRAW);
```

Comment:
If player disconnects than opponent should be winner, if we give result as draw, than player with lower rating gets advantage as he will get more points according to our rating system.
This way lower rating player will keep disconnecting and getting more points.

Reply:
Done now the opponent of disconnected player wins.